gammacat/webpage.py

```diff
@@ -44,8 +45,26 @@ def run(self):
         log.info('Make webpage ...')
         self.make_source_list_page()
         self.make_source_detail_pages()
+        self.make_publication_list()
         self.copy_data()
 
+    def make_publication_list(self):
+        publications = []
+        for x in (Path(gammacat_info.out_path / 'data')).iterdir():
+            publications.append((x.parts[len(x.parts)-1]).replace('%26', '&'))
```

Comment:
Is there a better way to decode the %26 to a &?

Reply:
Yes, there is a helper function to quote and unquote the URL:
https://gamma-cat.readthedocs.io/contribute/details.html#reference-identifiers
Since we do that already when building the index files, here you should never have to do it.
But if so (e.g. to make a web URL), please use that helper function again and not raw string manipulation.